contracts/package.json

```
@@ -48,6 +48,7 @@
     "test:pollFactory": "pnpm run test ./tests/PollFactory.test.ts",
     "test:subsidy": "pnpm run test ./tests/Subsidy.test.ts",
     "test:eas_gatekeeper": "pnpm run test ./tests/EASGatekeeper.test.ts",
+    "test:hatsGatekeeper": "pnpm run test ./tests/HatsGatekeeper.test.ts --network hardhat",
```

And return --network hardhat for test command

```suggestion
    "test:hatsGatekeeper": "pnpm run test ./tests/HatsGatekeeper.test.ts",
```